syntax = "proto3";

import "findit.proto";

package findit;

// Schema for a compile analysis event.
message CompileAnalysisCompletionEvent {
  // Base analysis info.
  Analysis analysis_info = 1;

  // Build failure type.
  repeated string failed_build_rules = 10;
}